source/wrangling.md

```diff
@@ -1032,53 +1046,61 @@ tidy_lang.loc[
 ]
 ```
 
-### Using `loc[]` to select ranges of columns
-
-Suppose we wanted to select only the columns `language`, `region`,
-`most_at_home` and `most_at_work` from the `tidy_lang` data set. Using what we
-learned in the chapter on {ref}`intro`, we would pass all of these column names into the square brackets.
+In addition to simultaneous subsetting of rows and columns, `loc[]` has two
```

Comment:
@joelostblom I think what was missing before was an emphasis on why we would care about loc[]. I've written it here to reference two special abilities of loc beyond [] -- ranges and logical statements to select columns.

Reply:
I like this additional motivation!